tests/test_xarray.py

```diff
@@ -1,10 +1,27 @@
 import pathlib
+import typing as T
 
+import pytest
 import xarray as xr
 
 DATA_FOLDER = pathlib.Path(__file__).parent / "data"
 
 
+def assert_common_attributes(attrs: T.Dict[T.Hashable, T.Any]) -> None:
```

Comment:
In general I don't like external assert functions.

Can this me moved to a COMMON_ATTRIBUTES variable and assert explicitly based on that?

Reply:
Removed in 814c428410b889c2b5fb3907c8ca1ae9c37b0411